pkg/cmd/attestation/verify/verify.go

```diff
@@ -55,7 +55,9 @@ func NewVerifyCmd(f *cmdutil.Factory, runF func(*Options) error) *cobra.Command
 			By default, the verify command will attempt to fetch attestations associated
 			with the provided artifact from the GitHub API. If you would prefer to verify
 			the artifact using attestations stored on disk (c.f. the %[1]sdownload%[1]s command),
-			provide a path to the %[1]s--bundle%[1]s flag.
+			provide a path to the %[1]s--bundle%[1]s flag. Additionally, the command
```

Comment:
This help page is getting so wordy we'll soon have to provide a table of contents and some headers. Or link to a docs page.

Anyways here is a suggested rewording:

```
By default, the verify command will:
- only verify provenance attestations
- attempt to fetch relevant attestations via the GitHub API.

To verify other types of attestations, use the predicate-type flag.

To use your artifact's OCI registry instead of GitHub's API, use the --bundle-from-oci flag. For offline verification, using attestations stored on desk (c.f. the download command), provide a path to the --bundle flag.
```

Reply:
(Like, re-reading the whole text, I'm like, we could stand to reword the section on specifying the owner. I'm thinking we need to call out that you're verifying the _identity_ of the signer, and maybe describe the reusable workflow options there instead of at the bottom – but this is a problem for another PR)